bitrise.yml

```diff
@@ -13,23 +13,22 @@ stages:
     - determine_apps_affected: {}
   build_applications:
     workflows:
-    - configure_build_fennec:
+    - firefox_configure_build:
         run_if: '{{getenv "BUILD_FIREFOX_IOS" | eq "true"}}'
-    # - configure_build_focus:
+    # - focus_configure_build:
     #     run_if: '{{getenv "BUILD_FOCUS_IOS" | eq "true"}}'
   run_tests:
     workflows:
-    - build_and_test_1_SmokeTest2:
+    - build_and_test_1_SmokeTest1:
```

Comment:
💯 Thanks for renaming this workflow. `test_1` and `SmokeTest2` has been inconsistent here.